src/Mod/Sketcher/App/Sketch.cpp

```diff
@@ -1797,6 +1797,20 @@ int Sketch::addConstraint(const Constraint *constraint)
                                         constraint->Second,constraint->SecondPos,
                                         c.value,c.driving);
         }
+        else if (constraint->FirstPos == PointPos::none &&
+            constraint->SecondPos == PointPos::none &&
+            constraint->Second != GeoEnum::GeoUndef &&
+            constraint->Third == GeoEnum::GeoUndef) {        // circle to circle, circle to arc, etc.
+
+            c.value = new double(constraint->getValue());

```

Comment:
@abdullahtahiriyo do you know if this heap allocations are correctly freed ?

Reply:
Two lines below, they are added to FixedParameters or Parameters. These are freed in clear().